cmd/fyne_demo/FyneApp.toml

```diff
@@ -3,10 +3,16 @@
   Name = "Fyne Demo"
   ID = "io.fyne.demo"
   Version = "2.3.0"
-  Build = 10
+  Build = 11
 
 [Development]
-  HelperText = "This binary was built with debug symbol"
+  HelperText = "This binary was built with debug symbols"
 
 [Release]
-  HelperText = "This binary was built without debug symbol"
+  HelperText = "This binary was built without debug symbols"
+
+[LinuxAndBSD]
+  GenericName = "Fyne Demo"
```

Comment:
What is generic name, and how often should it differ from the Name specified in the details section above?

Reply:
For Rymdport, I have "File Transfer" as GenericName. Just like Firefox has "Web Browser". I hope that helps.

Reply:
Ah yea thanks. I guess maybe it should be "demo app" or "developer tools" so it's clearer why they are different in this example?

Reply:
That's a good point. I'll get that fixed

Reply:
Perhaps "Toolkit Demo" even?

Reply:
Sounds good